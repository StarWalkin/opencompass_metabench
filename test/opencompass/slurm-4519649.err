slurmstepd: error: *** JOB 4519649 ON gpub089 CANCELLED AT 2024-08-18T00:08:54 ***
