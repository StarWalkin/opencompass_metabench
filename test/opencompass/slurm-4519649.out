/scratch/bcwg/Metabench/anaconda/envs/opencompass/bin/python3
